Makefile

```diff
@@ -60,12 +60,14 @@ check:
 	sudo cp ./testdata/database_config.json ${DBDIR}
 	sudo mkdir -p /usr/models/yang || true
 	sudo find $(MGMT_COMMON_DIR)/models -name '*.yang' -exec cp {} /usr/models/yang/ \;
-	-$(GO) test -mod=vendor $(BLD_FLAGS) -v github.com/Azure/sonic-telemetry/gnmi_server
-	-$(GO) test -mod=vendor $(BLD_FLAGS) -v github.com/Azure/sonic-telemetry/dialout/dialout_client
+	-sudo $(GO) test -v github.com/Azure/sonic-telemetry/sonic_db_config
+	-sudo $(GO) test -mod=vendor $(BLD_FLAGS) -v github.com/Azure/sonic-telemetry/gnmi_server
+	-sudo $(GO) test -mod=vendor $(BLD_FLAGS) -v github.com/Azure/sonic-telemetry/dialout/dialout_client
 
 clean:
 	$(RM) -r build
 	$(RM) -r vendor
+	$(RM) -r ${DBDIR}
```

Comment:
>DBDIR [](http://example.com/codeflow?start=12&length=5)

this is created by `sudo`, could you clean without `sudo`?

Reply:
yes updated and needed in Azure Pipeline setup. In local docker it was fine as running as `root`  so got missed.